validation/src/main/resources/application.conf

```diff
@@ -1,4 +1,24 @@
 hmda.validation.quality {
+  Q001 {
+    loan.amount=1000
+    loan.amount=${?Q001_LOAN_AMOUNT}
+    incomeMultiplier=5
+    incomeMultiplier=${?Q001_incomeMultiplier}
```

Comment:
Naming convention for environment variables would be `Q001_INCOME_MULTIPLIER`

... though I'm curious whether the multiplier is likely to change in the next few years. @lesgou or @debseidner , when edits like [Q001](https://github.com/cfpb/hmda-platform/issues/184) say "loan amount should be < 5 times the income", is the 5 likely to change in future years?